internal/nightfallconfig/nightfall_config.go

```diff
@@ -10,19 +10,23 @@ import (
 	nightfallAPI "github.com/nightfallai/nightfall_go_client/generated"
 )
 
+const defaultMaxNumberRoutines = 30
```

Comment:
why 30?

Reply:
30 is just an arbitrary number I chose. When we where talking with Rohan he thought that 25-30 go routines seemed like an appropriate default number.

This can be changed to anything we feel it appropriate.